packages/server/src/script/loadEmailTesting.ts

```diff
@@ -0,0 +1,127 @@
+import { sampleStateInfo } from '../service/letter'
+import { contactRecords } from '../service/contact'
+import { makeClient, httpConnector } from '@tianhuil/simple-trpc/dist/client'
+import { IVbmRpc, processEnvOrThrow } from '../common'
+import fetch from 'node-fetch'
+
+const serverUrl = processEnvOrThrow('REACT_APP_SERVER')
+const timeout = parseInt(processEnvOrThrow('REACT_APP_TIMEOUT'))
+
+export const client = makeClient<IVbmRpc>(
+  httpConnector(serverUrl, { timeout, fetch })
+)
+
+const fakeEmail = 'fake_voter@votebymail.io'
+
+interface Result {
+  success: number
+  failures: number
+}
+
+/**
+ * Asynchronously waits for a array full of promises to build, the promises
+ * inside the array might not be fulfilled yet--this function only makes sure
+ * the array is filled to the desired size.
+ *
+ * @param size The size of the array
+ * @param promise Function returning a promises to fill the array
+ * @param interval The amount of time in ms before pushing the next item
+ */
+const fillPromiseArray = async <T>(size: number, promise: () => Promise<T>, interval?: number): Promise<Promise<T | void>[]> => {
```

Comment:
This function should return `Promise<Array<T>>` -- you should use `Promise.all`

Reply:
The reason I didn't use `Promise.all` on `fillPromiseArray` is because if we do use `Promise<T[]>` some batches could take more than one second to fill because we'd have to await for all promises to be resolved in the return statement. Added this explanation as a comment.